src/vs/workbench/contrib/debug/browser/breakpointWidget.ts

```diff
@@ -130,13 +132,15 @@ export class BreakpointWidget extends ZoneWidget implements IPrivateBreakpointWi
 	}
 
 	private get placeholder(): string {
+		const acceptString = this.keybindingService.lookupKeybinding(AcceptBreakpointWidgetInputAction.ID)?.getLabel() || '';
+		const closeString = this.keybindingService.lookupKeybinding(CloseBreakpointWidgetCommand.ID)?.getLabel() || '';
```

Comment:
Should this fall back to `Enter` and `Esc` if `lookupKeybinding` returns undefined for any reason?

Reply:
I guess so